frontend/src/Student/components/ShowCard.js

```diff
@@ -46,7 +105,14 @@ ShowCard.propTypes = {
     id: PropTypes.string,
     name: PropTypes.string,
     entryCap: PropTypes.number,
-    entries: PropTypes.array,
+    entries: PropTypes.arrayOf(PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      title: PropTypes.string.isRequired,
+      __typename: PropTypes.oneOf(['Photo', 'Video', 'OtherMedia']).isRequired,
```

Comment:
@billdybas @chtinahow  if this looks easy/reliable enough we should totally just throw away my entry PR.

Reply:
I'd rather not use `__typename` for this. It's Apollo-specific, may change, and is really supposed to be used for internal caching. I'd keep the PR you have.